eng/pipelines/templates/variables/image.yml

```diff
@@ -2,20 +2,20 @@
 
 variables:
   - name: LINUXPOOL
```

Comment:
With the new pattern we might be able to remove these variables and just have one for each of the different architectures. That can be done as a follow-up though when we are thinking about applying this to other repos.

cc @benbp